src/tokens/aoTokens/ao.ts

```diff
@@ -524,7 +527,9 @@ export const sendAoTransferKeystone = async (
           name: "Recipient",
           value: recipient
         },
-        { name: "Quantity", value: amount }
+        { name: "Quantity", value: amount },
+        { name: "Client", value: "ArConnect" },
+        { name: "Client-Version", value: browser.runtime.getManifest().version }
```

Comment:
```suggestion
        { name: "Signing-Client", value: "ArConnect" },
        { name: "Signing-Client-Version", value: browser.runtime.getManifest().version }
```

Same as previous comment